packages/sdk/src/index.js

```
@@ -1956,19 +1938,14 @@ export async function createLivepeerSDK(
      * // }
      */
     async withdrawStake(
-      unbondingLockId: string,
+      unbondingLockId = null,
```

I think we shouldn't set up the default to be `null` as that would lead to failure  anyway. 
What do you think about considering a null check and fail early instead?

---

Agreed. I now perform a null check and throw an error if argument is null.

---

Also here it's unfortunate that I can't see the in-between diff between that review and now, only the end-result. Related to https://github.com/livepeer/livepeerjs/pull/570#issuecomment-590313254